api/src/main/java/ca/bc/gov/educ/eas/api/model/v1/AssessmentEntity.java

```diff
@@ -46,4 +47,9 @@ public class AssessmentEntity {
     @PastOrPresent
     @Column(name = "UPDATE_DATE", nullable = false)
     private LocalDateTime updateDate;
+
+    @EqualsAndHashCode.Exclude
+    @ToString.Exclude
+    @OneToMany(mappedBy = "assessmentEntity", fetch = FetchType.EAGER, cascade = CascadeType.ALL, targetEntity = AssessmentStudentEntity.class)
+    Set<AssessmentStudentEntity> students;
```

Comment:
It would be better to remove this bi-directional relationship and run operations independently i.e. delete the students using it's own entity and then run this.